data/docs/gcp-monitoring/gke-tracing.mdx

```diff
@@ -0,0 +1,1190 @@
+
+<figure data-zoomable align="left">
+    <img
+        src="/img/docs/gcp-monitoring/gke-tracing/self-hosted-application-level.webp"
+        alt="Application-level traces in SigNoz"
+    />
+<figcaption>
+<i>
+Application-level traces in SigNoz
+</i>
+</figcaption>
+</figure>
+
+## Troubleshooting
+
+If you encounter any issues while setting up logging and metrics for your GKE cluster, follow these troubleshooting steps:
+
+<Tabs>
+<TabItem value="self-hosted-on-gke-cluster" label="For Self-hosted SigNoz in GKE Cluster" default>
+
+1. Confirm that the necessary Kubernetes resources are created:
+
+    ```sh
+    kubectl get pods,services,configmaps -n signoz
+    ```
+
+    *Check if the required pods, services, and config maps are running and in a healthy state.*
+
+2. Check the logs of the OpenTelemetry Collector:
+
+    ```sh
+    kubectl logs -f -n signoz -l app.kubernetes.io/component=otel-agent
+    ```
+
+    Review the logs for any error messages or indications of misconfiguration.
+
+3. Verify the rendered configuration:
+
+    ```sh
+    kubectl get cm/kubelet-otel-k8s-infra-otel-agent -n signoz -o yaml
+    ```
+
+    Ensure the configuration matches your expected settings and the OpenTelemetry Collector endpoint.
+
+4. Confirm that the necessary Kubernetes resources are created:
+
+    ```sh
+    kubectl get pods,services,configmaps -n platform
+    ```
+
+    Check if the required pods, services, and config maps are running and in a healthy state.
+
+5. Check if all the Pixie deployments are in the correct order:
+    * Check all the configurations and deployments are in the correct order so that Pixie can forward all the scraped traces to the respective endpoint.
+
+
+</TabItem>
+<TabItem value="self-host-signoz-in-vm" label="For Self-hosted SigNoz in a VM">
+
+1. Check the logs of the OpenTelemetry Collector:
+
+    ```sh
+    kubectl logs -f -n signoz -l app.kubernetes.io/component=otel-agent
+    ```
+
+    Review the logs for any error messages or indications of misconfiguration.
+
+2. Verify the rendered configuration:
+
+    ```sh
+    kubectl get cm/kubelet-otel-k8s-infra-otel-agent -n signoz -o yaml
+    ```
+
+    Ensure the configuration matches your expected settings and the OpenTelemetry Collector endpoint.
+
+3. Confirm that the necessary Kubernetes resources are created:
+
+    ```sh
+    kubectl get pods,services,configmaps -n signoz
+    ```
+
+    *Check if the required pods, services, and config maps are running and in a healthy state.*
+
+4. Verify network connectivity:
+
+    * Ensure that the GKE cluster has network access to the SigNoz ingestion endpoint (Self-Hosted SigNoz VM IP:4317).
+    * Check if there are any network security groups or firewalls blocking the required ports.
+
+5. Check the SigNoz Self-Host VM
+    * Confirm that all the SigNoz related services are running properly on the VM or not.
+
+6. Check if all the Pixie deployments are in the correct order:
```

Comment:
AFAIK, we were not running pixie on VM, so it is not needed for app tracing. Can you confirm once with sumedh as well?

Reply:
@sum3dh

Reply:
Yes, we can skip this part for app tracing.

Reply:
added a note